src/modules/launcher/Plugins/Microsoft.PowerToys.Run.Plugin.Service/Helpers/ServiceHelper.cs

```diff
@@ -22,39 +23,58 @@ public static class ServiceHelper
         public static IEnumerable<Result> Search(string search, string icoPath, PluginInitContext context)
         {
             var services = ServiceController.GetServices();
+            IEnumerable<ServiceController> serviceList = new List<ServiceController>();
 
-            return services
-                .Where(s => s.DisplayName.StartsWith(search, StringComparison.OrdinalIgnoreCase) || s.ServiceName.StartsWith(search, StringComparison.OrdinalIgnoreCase) || GetResultTitle(s).StartsWith(search, StringComparison.OrdinalIgnoreCase))
-                .Select(s =>
+            if (search.StartsWith(Resources.wox_plugin_service_status + ":", StringComparison.CurrentCultureIgnoreCase))
+            {
+                // allows queries like 'status:running'
+                serviceList = services.Where(s => GetLocalizedStatus(s.Status).Contains(search.Split(':')[1], StringComparison.CurrentCultureIgnoreCase)).ToList().OrderBy(x => (int)x.Status);
```

Comment:
Why order by status in the end? Leave it ordered alphabetically should make more sense imho

Reply:
As this is filtering by state and I thought the it makes sense to do this.

Btw. I found that it isn't ordered alphabetical. It's ordered like ServiceController does and this isn't alphabetical or not by display name. Should we fix this at all with ordering by display name?

Reply:
ok. In that case let's leave it as it is here. If we decide to order everything alphabetically, we'll change it here as well

Reply:
i think we can order alphabetical by display name in line 25 where we get the services

Reply:
ok, I just checked the results order.. it's pretty messy :) let's order alphabetically by display name. line 25 should be fine